indexing-service/src/main/java/org/apache/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -924,6 +924,11 @@ public CommandListBuilder addSystemProperty(String property, boolean value)
       return addSystemProperty(property, String.valueOf(value));
     }
 
+    public CommandListBuilder addSystemProperty(String property, Object value)
```

Comment:
This new method feels unnecessary.
Maybe we can use the approach used in #16281 instead. 

Since the original author has not responded on that PR, we can proceed with this one.

Reply:
Done.